library/newrelic-usage/dashboards/nr_usage_apof.json

```
@@ -0,0 +1,350 @@
+{
+  "name": "New Relic APoF Consumption",
+  "description": "",
+  "pages": [
+    {
+      "name": "New Relic APoF Consumption",
+      "description": "",
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 6
+          },
+          "title": "",
+          "rawConfiguration": {
+            "text": "# Understanding your usage data in New Relic One\n\nThese queries include common scenarios for counting usage and estimating cost across New Relic One. \n\nTo set usage alerts to manage spend, visit our [docs page](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-users/usage-queries-alerts).\n\nFor month-to-date usage averages and cost, visit the [Usage UI](https://one.newrelic.com/launcher/usage-ui.launcher) in New Relic One.\n\nTo understand data ingest by event, type, or integration source, explore the [Data Management Hub](https://one.newrelic.com/launcher/data-management-nerdlets.data-management-launcher).\n\n"
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 7,
+            "row": 1,
+            "height": 3,
+            "width": 3
+          },
+          "title": "Latest Full User count",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM NrConsumption SELECT latest(FullUsers) since 1 day ago"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 10,
+            "row": 1,
+            "height": 3,
+            "width": 3
+          },
+          "title": "This Month: Full User average",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "from NrMTDConsumption SELECT latest(usersBillable) where metric= 'FullUsers' since this month"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.bullet"
+          },
+          "layout": {
+            "column": 1,
+            "row": 4,
+            "height": 3,
+            "width": 5
+          },
+          "title": "This Month: Usage vs. Annual Pool of Funds",
+          "rawConfiguration": {
+            "limit": 8251,
```

Since this is meant to reflect a customer's monthly APoF allocation, I recommend setting it to 0 in the template and updating the widget title to indicate that the user must update it to reflect their own allocation.
```suggestion
            "limit": 0,
```